README.md

```diff
@@ -16,14 +16,34 @@ Usage
 1. [Install Ansible](http://docs.ansible.com/ansible/intro_installation.html) on your local computer and verify that it works
     - ansible --version
     - note: on MacOS Ansible might not be added to your PATH in bash_profile, see [2nd answer in this thread](https://stackoverflow.com/questions/35898734/pip-installs-packages-successfully-but-executables-not-found-from-command-line/35899029) on how to fixed that.
-1. [Initialize a memory card](https://www.raspberrypi.org/documentation/installation/installing-images/) (on your local computer) with the latest [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/) with [Etcher](https://www.balena.io/etcher/) or from the command line:
-    - `diskutil list`
-    - `diskutil unmountDisk /dev/<disk#>`
-    - `sudo dd bs=1m if=<your image file>.img of=/dev/<disk#>`
-1. [Enable ssh](https://www.raspberrypi.org/blog/a-security-update-for-raspbian-pixel/) for Raspbian headless operation (while memory card is still attached to your local computer)
-    - `touch /Volumes/boot/ssh`
-1. Insert the memory card in the to-be-provisioned Raspberry Pi and connect the it to the local network.
-1. Clone this git repository on your local computer (if you haven't already done so)
+1. Install Raspberry OS Lite on an SD-card with your local computer.
+    - Download the [Raspberry Pi Imager] (https://www.raspberrypi.org/software/) and install the OS (the tool will download the latest image)
+    - Alternatively you can download the image yourself and write this to the SD-card with a tool like Belana Etcher or directly from the command line with the following commands:
```

Comment:
typo: Balena. That could also be a link.

Reply:
Will correct this.